### ISSUE TITLE
Chaoss Brand guide Documentation

### ISSUE THREAD

**Issue by Toriasdesign** [chaoss/chaoss-africa#22]
The assets from the brand design will arranged in a PDF document format, summarizing the brand details on file.

**Comment by Busayo-ojo** [comment]
@iamkingsleey Has this been worked on?
Do you know if new contributors can still work on this?

**Comment by iamkingsleey** [comment]
Nope, @Busayo-ojo

**Comment by Mayaleeeee** [comment]
Is this still open?
cc @iamkingsleey @Busayo-ojo

**Comment by iamkingsleey** [comment]
Yes, it is @Mayaleeeee. I replied to your comment on another issue connected to this.

**Comment by Piyush0000** [comment]
Hi @iamkingsleey   

I’d like to work on this issue.  
I can help create a structured and clear brand guide document (PDF) summarizing CHAOSS branding assets.  

Could you please assign this to me?  
Thank you!
